kiali-server/templates/_helpers.tpl

```diff
@@ -32,6 +32,17 @@ Create chart name and version as used by the chart label.
 {{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" }}
 {{- end }}
 
+{{/*
+Identifies the log_level with the old verbose_mode and the new log_level considered.
+*/}}
+{{- define "kiali-server.logLevel" -}}
+{{- if .Values.deployment.verbose_mode -}}
+{{- tpl .Values.deployment.verbose_mode . -}}
```

Comment:
I'm just curious why you used "tpl" here but not for using the log_level value below in the else clause.

Reply:
I copied the logic from one of my other projects where the first value could be a Helm argument as well. Thats why `tpl` would render the `verbose_mode` like any other Helm argument in a template in the context of the second argument `.`. Actually we do not need this in this use-case, I am removing the `tpl` function.